src/providers/twitch/api/Helix.cpp

```diff
@@ -8,6 +8,19 @@
 
 namespace chatterino {
 
+HelixUserList::HelixUserList() {
+    users = std::unordered_set<QString>();
+}
+
+void HelixUserList::AddUsersFromResponse(const QJsonObject &response) {
+    for (const auto jsonStream : response.value("data").toArray()) 
+    {
+        auto user = jsonStream.toObject().value("user_login").toString();
+        users.insert(user);
```

Comment:
Use `this->` when accessing member variable

so `this->users.insert(user);` instead of `users.insert(user);`

Reply:
class deleted